File: 1832. Check if the Sentence Is Pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        map<char,int> mp;
        for(char a='a';a<='z';a++) {
            mp[a]=0;
        }
        for(int i=0;i<sentence.length();i++) {
            mp[sentence[i]]++;
        }
        for(auto const& x: mp) {
            if(x.second == 0) return false;
        }
        return true;
    }
};

File: 1832. Check if the Sentence Is Pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        dict={}
        c='a'
        i=25
        s='abcdefghijklmnopqrstuvwxyz'
        for i in s:
            dict[i]=0
    
        for i in sentence:
            dict[i] = 1
            
        for i in list(dict.values()):
            if i==0 :
                return 0;
        return 1;    
